snapcraft/internal/common.py

```diff
@@ -124,8 +126,8 @@ def is_snap() -> bool:
     return is_snap
 
 
-def is_docker_instance() -> bool:
-    return os.path.exists(_DOCKERENV_FILE)
+def is_in_container() -> bool:
```

Comment:
can you call this differently, the word container is a bit vague and certainly is not relevant to LXD containers

Reply:
Ephemeral LXD containers behave the same. `def container_mode() -> bool` is ok? With explanation of what container mode is.

Reply:
Renamed to `is_process_container` to be consistent with `is_snap` above and to distinguish between LXD full OS containers.